source/features/clean-rich-text-editor.tsx

```diff
@@ -2,5 +2,25 @@ import './clean-rich-text-editor.css';
 import * as pageDetect from 'github-url-detection';
 
 import features from '../feature-manager.js';
+import select from 'select-dom';
+
+function hideToolTip(): void {
+    for (const textarea of select.all(`
+        input[id="issue_title"], 
+        textarea[id="issue_form_repro"], 
+        textarea[id="issue_form_description"], 
+        input[id="issue_form_browser", 
+        textarea[id="new_comment_field"]
+    `)) {
+        textarea.title = '';
+    }
+}
 
 void features.addCssFeature(import.meta.url, [pageDetect.hasRichTextEditor]);
```

Comment:
This should be removed. In the new init function, it should just add the same class to the body/html. Refer to the source of addCssFeature to see what it does